samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/QueryRelatedFeatures.razor

```diff
@@ -13,7 +13,8 @@
 <MapView Class="map-view"
          OnClick="OnClick"
          Longitude="-98.5795" Latitude="39.8282"
-         Zoom="3">
+         Zoom="3"
+         PopupEnabled="false">
```

Comment:
I answered my own question on what is the default value when I read the Summary in `QueryRelatedFeatures`